Bugfix from upstream. See:
    <http://www.zsh.org/mla/workers/2011/msg00237.html>

Index: pkg-zsh/Src/lex.c
===================================================================
--- pkg-zsh.orig/Src/lex.c	2011-02-24 22:14:59.000000000 +0100
+++ pkg-zsh/Src/lex.c	2011-02-24 22:15:04.000000000 +0100
@@ -1823,7 +1823,7 @@
 	    int zp = lexflags;
 
 	    gotword();
-	    if (zp == 1 && !lexflags) {
+	    if ((zp & LEXFLAGS_ZLE) && !lexflags) {
 		if (zshlextext == copy)
 		    zshlextext = tokstr;
 		return 0;
